src/sentry/release_health/metrics.py

```diff
@@ -458,165 +447,105 @@ def run_sessions_query(
     ) -> SessionsQueryResult:
         return run_sessions_query(org_id, query, span_op)
 
-    # TODO RaduW 23.Jan.2023 Reimplement when the Metrics Layer adds support for timestamps
     def get_release_sessions_time_bounds(
         self,
         project_id: ProjectId,
         release: ReleaseName,
         org_id: OrganizationId,
         environments: Optional[Sequence[EnvironmentName]] = None,
     ) -> ReleaseSessionsTimeBounds:
-        """
-        Note: this is copied as is from the old metrics implementation, it does NOT use the Metrics Layer
-        """
-        select: List[SelectableExpression] = [
-            Function("min", [Column("timestamp")], "min"),
-            Function("max", [Column("timestamp")], "max"),
-        ]
 
-        try:
-            where: List[Condition] = [
-                Condition(Column("org_id"), Op.EQ, org_id),
-                Condition(Column("project_id"), Op.EQ, project_id),
-                Condition(
-                    Column(resolve_tag_key(USE_CASE_ID, org_id, "release")),
-                    Op.EQ,
-                    resolve_weak(USE_CASE_ID, org_id, release),
-                ),
-                Condition(
-                    Column("timestamp"), Op.GTE, datetime(2008, 5, 8)
-                ),  # Date of sentry's first commit
-                Condition(
-                    Column("timestamp"), Op.LT, datetime.now(timezone.utc) + timedelta(seconds=10)
-                ),
-            ]
+        projects, org_id = self._get_projects_and_org_id([project_id])
 
-            if environments is not None:
-                env_filter = resolve_many_weak(USE_CASE_ID, org_id, environments)
-                if not env_filter:
-                    raise MetricIndexNotFound()
+        select = [
+            MetricField(metric_mri=SessionMRI.SESSION.value, alias="min1", op="min_timestamp"),
+            MetricField(metric_mri=SessionMRI.SESSION.value, alias="max1", op="max_timestamp"),
+            MetricField(metric_mri=SessionMRI.RAW_DURATION.value, alias="min2", op="min_timestamp"),
```

Comment:
Can we find a better alias name? Or if we don't need it, we may be avoiding it.

Reply:
Ok, will think of some better names, I prefer aliases since I find the automatic name a bit difficult to link back to the original field definition, and it also gets large when using operators.